packages/core/test/request-queue/adding-the-same-request-should-not-call-the-api.test.ts

```diff
@@ -27,7 +27,11 @@ describe('RequestQueue#addRequest should not call the API if the request is alre
 
         expect(clientSpy).toHaveBeenCalledTimes(1);
 
-        await requestQueue.markRequestHandled({ id: requestData.requestId, uniqueKey: requestData.uniqueKey } as any);
+        await requestQueue.markRequestHandled({
```

Comment:
isn't this technically breaking? why do you need the URL now?

Reply:
According to the types we always needed the url. The test _used_ to work because we checked that the request was in progress before marking it as handled (which I removed because that makes no sense to check), which didn't need the url 👀

Reply:
oh right, overlooked the `as any` that was there before, ok